### PR TITLE
Case Study 3 - Page 8.html created

### PR DESCRIPTION
**Pull request by megha994**
New page Page 8.html created for Case Study 3. 
Changes made to 6 files.

### CODE REVIEW

File: styles/components/_icons.scss [hunk 1]
```diff
@@ -9,113 +8,125 @@
...
-    content: '\F444';
+  content: "\F444";
 }
 .icon-mail::before {
-    content: '\F32C';
+  content: "\F32C";
+}
+.icon-person::before {
+  content: "\F8F6";
+  color: #000000 !important;
```
Comment (ericnjanga) [inline styles/components/_icons.scss]: @megha994   Remove the color here. The color should be assigned in the parent component.

File: styles/components/_icons.scss [hunk 2]
```diff
@@ -9,113 +8,125 @@
...
+}
+.icon-graph-up-arrow::before {
+  content: "\F673";
+  color: #000000 !important;
```
Comment (ericnjanga) [inline styles/components/_icons.scss]: Remove the color here. The color should be assigned in the parent component.

File: styles/components/_icons.scss [hunk 3]
```diff
@@ -9,113 +8,125 @@
...
+}
+.icon-thermometer-sun::before {
+  content: "\F5D1";
+  color: #000000 !important;
```
Comment (ericnjanga) [inline styles/components/_icons.scss]: Remove the color here. The color should be assigned in the parent component.

File: styles/components/_icons.scss [hunk 4]
```diff
@@ -9,113 +8,125 @@
...
+}
+.icon-award-1::before {
+  content: "\F154";
+  color: #000000 !important;
```
Comment (ericnjanga) [inline styles/components/_icons.scss]: Remove the color here. The color should be assigned in the parent component.

File: styles/components/_icons.scss [hunk 5]
```diff
@@ -9,113 +8,125 @@
...
-        content: '\F1C5';
-    }
-    &.icon-link::before {
-        content: '\F470';
-    }
+  &.icon-pin::before {
+    content: "\F3E7";
+  }
+  &.icon-pdf::before {
+    color: #313d2f !important;
```
Comment (ericnjanga) [inline styles/components/_icons.scss]: Remove the color here. The color should be assigned in the parent component.

File: styles/components/_card.scss [hunk 6]
```diff
@@ -1,56 +1,74 @@
...
-            margin-bottom: 0!important;
-        }
-    }
-    .halton-card.card-icon::before {
-        color: var(--halton-clr-card-icon)!important;
-        top: 10px;
-    }
-}
\ No newline at end of file
+.card-community {
```
Comment (ericnjanga) [inline styles/components/_card.scss]: @megha994  Eliminate this whole class completely. We don't need it.

File: _pages/8.html [hunk 7]
```diff
@@ -0,0 +1,404 @@
...
+                    <!------------------------?????---------------------------------->
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="bottom-section-spacer-sm">
+            <h2 id="01" class="modified-heading bottom-section-spacer-sm">
+              <span class="modified-heading">
```
Comment (ericnjanga) [inline _pages/8.html:166]: @megha994    Why wrapping this inside a <span class="modified-heading">??

File: _pages/8.html [hunk 8]
```diff
@@ -0,0 +1,404 @@
...
+          </div>
+        </div>
+      </div>
+
+      <!--------------->
+
+      <div class="row marg rbeh">
+        <div class="col-lg-6 col-md-6">
+          <div
+            class="card-community card-1 halton-card card card-icon icon-person rbeh-item"
```
Comment (ericnjanga) [inline _pages/8.html]: Redo this card using the same markup already available on page 3.html:
--> Don't worry about the icons or the cards background color for now
--> Do the same for all the 4 cards


<div class="halton-card card card-icon icon-mail rbeh-item">
                                <div class="card-body">
                                    <h3 class="card-title">Online</h3>
                                    <p>
                                        Send an email to <a
                                            href="mailto:smallbusiness@halton.ca">smallbusiness@halton.ca </a>.
                                    </p>
                                    <p>Carefully consider the use of electronic communication for sensitive
                                        information. Halton
                                        Region will use reasonable means to protect your information, however, due
                                        to the
                                        inherent risks of electronic communications, confidentiality cannot be
                                        guaranteed.</span></p>
                                </div>
                            </div>

**Review by ericnjanga — CHANGES_REQUESTED** [review]
Please address my comments.

File: _pages/8.html (hunk 7 above)
Comment (megha994) [reply _pages/8.html:166]: I wanted to maintain the heading as h2 but apply the styles of the modified-heading so I placed inside a <span>. However I had to remove the modified-heading class from the h2 tag.

**Pull request closed without merging**